pkg/cloud/libvirt/client/network.go

```diff
@@ -16,8 +17,15 @@ const (
 	netModeNat      = "nat"
 	netModeRoute    = "route"
 	netModeBridge   = "bridge"
+	workerIPCidr    = 51
 )
 
+// Leases contains list of Lease
```

Comment:
Not the most useful comment. :) `Contains list of DHCP leases` would be better IMO.

Reply:
Done